<launch>

    <node name="usb_cam_node" pkg="usb_cam" type="usb_cam_node" output="screen">
        <remap from="/usb_cam_node/image_raw" to="/camera/rgb/image_raw" />
    </node>

    <!-- <include file="$(find mediapipe_track)/launch/locker_human.launch"/> -->

    <include file="$(find mediapipe_track)/launch/rviz.launch"/>

</launch>